Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -4593,4 +4593,38 @@ M  END)CTAB";
       CHECK(m->getBondWithIdx(0)->getIsAromatic());
     }
   }
-}
\ No newline at end of file
+}
+
+TEST_CASE(
+    "Github #5165: issue with V3000 SD files containing enhanced "
+    "stereochemistry information") {
+    
+    SECTION("as reported") {
+        std::string rdbase = getenv("RDBASE");
+        std::string fName =
+          rdbase + "/Code/GraphMol/FileParsers/test_data/mol_with_enhanced_stereo_2_And_groups.sdf";
+        SDMolSupplier suppl(fName);
+        ROMol *mol = suppl.next();
```

Comment:
Since many things can go wrong at this point (bad environment variable, missing files, code changes which break parsing, etc.), it's always a good idea to confirm that you actually got a valid molecule:
```suggestion
        ROMol *mol = suppl.next();
        REQUIRE(mol);
```